deploy/template.yaml

```diff
@@ -12,9 +12,9 @@ Globals:
     Tracing: Active
     VpcConfig:
       SecurityGroupIds:
-        - sg-03a1efaf2fed58442
+        - sg-0baf479c53ac54f59
```

Comment:
ok to harcode this for now, but can we get this from an SSM param or a [mapping like in HKR ](https://github.com/alphagov/verify-hub-key-rotation/blob/6d2801dc8138e964617ac2980825450e4157a71c/template.yaml#L36)in future?